packages/sync/src/class-replicastore.php

```diff
@@ -884,6 +884,12 @@ public function get_terms( $taxonomy ) {
 	 * @return \WP_Term|\WP_Error Term object on success, \WP_Error object on failure.
 	 */
 	public function get_term( $taxonomy, $term_id, $is_term_id = true ) { // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
+
+		// Full Sync will pass false for the $taxonomy so a check for term_taxonomy_id is needed before ensure_taxonomy.
+		if ( 'term_taxonomy_id' === $is_term_id ) {
```

Comment:
The doc on line 883 must be incorrect/incomplete, supposedly `$is_term_id` is a bool rather than a string.

Reply:
yes its incomplete, downstream functionality passes "term_taxonomy_id" there isn't an implementation that uses false.